internal/cmd/password/password.go

```diff
@@ -107,6 +110,7 @@ func toPassword(password *ps.DatabaseBranchPassword) *Password {
 		Username:  password.Username,
 		Role:      password.Role,
 		RoleDesc:  toRoleDesc(password.Role),
+		Replica:   password.Replica,
```

Comment:
I wonder if we should do something like `Connection type: Primary / Replica` here instead of a boolean?

Reply:
Ooh, yeah!